doc/userguide/conf.py

```diff
@@ -143,6 +143,7 @@ def setup(app):
         else:
             app.add_stylesheet('css/suricata.css')
 else:
+    html_theme = 'sphinx_rtd_theme'
```

Comment:
Will want to cherry-pick this commit back to 6.0.x otherwise the theme will change for 6.0.14. Its a clean cherry-pick, want a new PR for it though?

Reply:
yes please